esp/scm/WsDeploy.ecm

```diff
@@ -568,7 +568,7 @@ ESPresponse [exceptions_inline, encode(0)] GetSummaryResponse
     string XmlStr;
 };
 
-ESPservice [requiredaccess("NONE"), exceptions_inline("xslt/exceptions.xslt")] WsDeploy
+ESPservice [autofeatureauth("NONE"), exceptions_inline("xslt/exceptions.xslt")] WsDeploy
```

Comment:
@garonsky I think we've talked about this before, but please confirm if it makes sense for wsdeploy not to perform feature level auth... thanks.